chain/datasource/datasource.go

```diff
@@ -275,6 +276,10 @@ func (t *DataSource) ExecutedAndBlockMessages(ctx context.Context, ts, pts *type
 	return value.(*lens.TipSetMessages), nil
 }
 
+func (t *DataSource) MinerLoad(store adt.Store, act *types.Actor) (miner.State, error) {
```

Comment:
Move the logic responsible for loading miners from the blockstore to an interface allowing tests to use [mocked miner state](https://github.com/filecoin-project/lily/pull/994/files#diff-7ae591e2ef4d70dd2ffde4567444db7658ef2ce912cc41bc5fca429353a2aa0a) for validation.